src/domains/event/dto/index.ts

```diff
@@ -0,0 +1,106 @@
+export type BlockEventType = 'NO_STATUS' | 'BLOCKED_BY' | 'BLOCKING_TO';
+export type EventInputArray<T extends EventInput> = T[];
+
+export enum TrickerBlockEventType {
+  NO_STATUS = 'NO_STATUS',
+  BLOCKED_BY = 'BLOCKED_BY',
+  BLOCKING_TO = 'BLOCKING_TO',
+}
+
```

Comment:
Specific LinearBlockTypeConvention and ActionType should not be within Event Domain